brianlong/www.validators.app
# 183982396 - track authorities for vote account

#### What's this PR do?
PR adds versioning of `authorized_withdrawer` in vote accounts. It also appends new attribute - `authorized_voters`

#### How should this be manually tested?
Please test using the `rails c`.

After any changes made to either `authorized_withdrawer` or `authorized_voters` the change history should be accessible via `account_authority_histories` table. Please verify it.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183982396)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
